src/main/java/lovespiritual.java

```diff
@@ -22,10 +23,40 @@ public static void main(String[] args) {
                 break;
             } else if (input.equalsIgnoreCase("list")) {
                 System.out.println(line);
+                System.out.println("Here are the tasks in your list:");
                 for (int i = 0; i < count; i++) {
-                    System.out.println((i + 1) + ". " + list[i]);
+                    String checkbox = status[i] ? "[X]" : "[ ]";
+                    System.out.println((i + 1) + "." + checkbox + " " + list[i]);
                 }
                 System.out.println(line);
+            } else if (input.startsWith("mark ")) {
+                String numberString = input.substring(5).trim();
+                int number = Integer.parseInt(numberString) - 1;
+                if (number >= 0 && number < count) {
+                    status[number] = true;
+                    System.out.println(line);
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println(" [X] " + list[number]);
+                    System.out.println(line);
+                } else {
+                    System.out.println(line);
+                    System.out.println("Invalid number. Please enter a valid number.");
+                    System.out.println(line);
+                }
+            } else if (input.startsWith("unmark ")) {
+                String numberString = input.substring(7).trim();
```

Comment:
The nested loop involves more than 1 level which may slightly violate the coding standard but it looks good